### PR TITLE
Fix warnings in project.json

### LINKED ISSUE #3038: Project.json warnings in build
After the schema change at #2493, there are several hundred warnings in the tree.

## Steps to reproduce
build the CLI tree

## Expected  behavior
Warnings only related to tests

## Actual behavior
Several hundred warnings 

## Environment data
`dotnet --info` output:

### PR DESCRIPTION
**Pull request by billwert**
PR #2493 introduced the new project.json schema. The tree has 124 files
with the old schema, which added several hundred warnings.

This change can't go in until PR #2864 does - it relies on those bug
fixes.

This fixes #3038

### CODE REVIEW

**Comment by billwert** [comment]
The test failures are due to missing #2864; not sure how y'all typically handle things like this.

**Comment by eerhardt** [comment]
@billwert - #2864 is now in.  Can you rebase and we can get this merged?

**Comment by billwert** [comment]
@eerhardt all good.

File: test/Microsoft.DotNet.Cli.Utils.Tests/project.json [hunk 1]
```diff
@@ -1,8 +1,17 @@
 {
   "version": "1.0.0-*",
-  "compilationOptions": {
+  "buildOptions": {
     "emitEntryPoint": true,
-    "keyFile": "../../tools/Key.snk"
+    "keyFile": "../../tools/Key.snk",
+    "copyToOutput": {
+      "include": [
+        "../../TestAssets/TestProjects/OutputStandardOutputAndError/*",
```
Comment (eerhardt) [inline test/Microsoft.DotNet.Cli.Utils.Tests/project.json]: I know you didn't design the format - but IMO this points out a design flaw.  How often do you want the "copyToOutput" files copied both on build and on publish?  Most of the time, I would think.  I would also want them in my "pack" as well ...

**Comment by eerhardt** [comment]
:shipit:

File: test/Microsoft.DotNet.Cli.Utils.Tests/project.json (hunk 1 above)
Comment (billwert) [inline test/Microsoft.DotNet.Cli.Utils.Tests/project.json]: agreed. and you're probably right that I should have added the pack option as well, to match the previous semantic.. @ajaybhargavb - thoughts on this overall? perhaps with msbuild coming it doesn't matter much.

**Pull request merged**